### PR TITLE
Server dev

### PR DESCRIPTION
**Pull request by AamirAlam**

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -97,19 +97,19 @@ export default defineNuxtConfig({
...
-    AWS_ACCESS_KEY_ID: process.env.AWS_ACCESS_KEY_ID_ECO,
-    AWS_SECRET_ACCESS_KEY: process.env.AWS_SECRET_ACCESS_KEY_ECO,
-    ROOT_BUCKET: process.env.ROOT_BUCKET,
-    IMAGE_UPLOAD_SIZE_LIMIT: process.env.IMAGE_UPLOAD_SIZE_LIMIT,
-    GIT_ACCESS_TOKEN: process.env.GIT_ACCESS_TOKEN,
-    JWT_SECRET: process.env.JWT_SECRET,
-    GITHUB_WEBHOOK_SECRET: process.env.GITHUB_WEBHOOK_SECRET,
+    mongodbUri: process.env.NUXT_MONGODB_URI,
+    awsAccessKeyIdEcho: process.env.NUXT_AWS_ACCESS_KEY_ID_ECO,
+    awsSecretAccessKeyEcho: process.env.NUXT_AWS_SECRET_ACCESS_KEY_ECO,
```
Comment (Ashar2shahid) [inline nuxt.config.ts:102]: Echo ?

File: nuxt.config.ts [hunk 2]
```diff
@@ -97,19 +97,19 @@ export default defineNuxtConfig({
...
-    mongodbURI: process.env.DEV_DB_URL,
-    AWS_ACCESS_KEY_ID: process.env.AWS_ACCESS_KEY_ID_ECO,
-    AWS_SECRET_ACCESS_KEY: process.env.AWS_SECRET_ACCESS_KEY_ECO,
-    ROOT_BUCKET: process.env.ROOT_BUCKET,
-    IMAGE_UPLOAD_SIZE_LIMIT: process.env.IMAGE_UPLOAD_SIZE_LIMIT,
-    GIT_ACCESS_TOKEN: process.env.GIT_ACCESS_TOKEN,
-    JWT_SECRET: process.env.JWT_SECRET,
-    GITHUB_WEBHOOK_SECRET: process.env.GITHUB_WEBHOOK_SECRET,
+    mongodbUri: process.env.NUXT_MONGODB_URI,
+    awsAccessKeyIdEcho: process.env.NUXT_AWS_ACCESS_KEY_ID_ECO,
```
Comment (Ashar2shahid) [inline nuxt.config.ts:101]: Echo ?

Comment (AamirAlam) [reply nuxt.config.ts:101]: name could be anything  , I used this it was already in config

Comment (Ashar2shahid) [reply nuxt.config.ts:101]: change it to awsSecretAccessKeyEco for consistency

File: nuxt.config.ts [hunk 3]
```diff
@@ -97,19 +97,19 @@ export default defineNuxtConfig({
...
+    imageUploadSizeLimit: process.env.NUXT_IMAGE_UPLOAD_SIZE_LIMIT,
+    gitAccessToken: process.env.NUXT_GIT_ACCESS_TOKEN,
+    jwtSecret: process.env.NUXT_JWT_SECRET,
+    githubWebhookSecret: process.env.NUXT_GITHUB_WEBHOOK_SECRET,
+    nodeEnv: process.env.NUXT_NODE_ENV,
+    dappRegistryProd: process.env.NUXT_DAPP_REGISTRY_PROD,
+    dappRegistryDev: process.env.NUXT_DAPP_REGISTRY_DEV,
     public: {
-      WEB3MODAL_PROJECT_ID: process.env.WEB3MODAL_PROJECT_ID,
+      projectId: process.env.NUXT_PUBLIC_PROJECT_ID,
```
[the remark this answers is not in the document]
Comment (AamirAlam) [reply nuxt.config.ts:112]: It was casing issue while parsing with previous name

**Pull request merged**